JASP-Engine/JASP/R/common.R

```diff
@@ -1021,7 +1021,7 @@ saveImage <- function(plotName, format, height, width){
 .imgToState <- function(imgobj) {
 
 	result <- list()
-	if (!is.list(imgobj))
+	if (!is.list(imgobj) || is.null(names(imgobj)))
```

Comment:
@TimKDJ can you please add a comment before the ```if``` statement (for future reference)?